extensions/blocks/membership-button/view.scss

```diff
@@ -0,0 +1,50 @@
+/* Additional styling to thickbox that displays modal */
+/* stylelint-disable selector-max-id */
+
+.jetpack-memberships-modal #TB_title {
+	border-radius: 4px 4px 0 0;
+}
+#TB_window.jetpack-memberships-modal {
+	border-radius: 4px;
+	background-color: #f9f9f9;
+	background-image: url( 'https://s0.wp.com/i/loading/loading-64.gif' );
+	background-repeat: no-repeat;
+	background-position: center;
+	bottom: 10%;
+	margin-top: 0 !important;
+	top: 10%;
+}
+
+.jetpack-memberships-modal #TB_iframeContent {
+	height: calc( 100% - 50px ) !important;
+}
+@media only screen and ( max-width: 480px ) {
+	#TB_window.jetpack-memberships-modal {
+		bottom: 0;
+		left: 0;
+		margin-left: 0 !important;
+		right: 0;
+		top: 0;
+		width: 100% !important;
+	}
+	.jetpack-memberships-modal #TB_iframeContent {
+		width: 100% !important;
+	}
+}
+
+.jetpack-memberships-modal #TB_iframeContent {
+	height: calc( 100% - 80px ) !important;
+}
+.jetpack-memberships-modal .TB_footer {
+	border-top: 1px solid #ddd;
+	color: #a5b2bd;
```

Comment:
We shouldn't use fixed colors like this. Please use either Gutenberg colour variables (by adding them on top of this file so that we can easily migrate them later to use Gutenberg sass files, [see example](https://github.com/Automattic/jetpack/blob/858b174e510ebcb59904cbeb0a89a072870d33e1/extensions/blocks/markdown/editor.scss#L1-L3)) or by using Muriel colour CSS variables listed here: https://github.com/Automattic/color-studio/blob/546e4ed57ec2426246049096fbd3064e1d4ecfb0/dist/color-properties.css

Reply:
@lessbloat any help here? :)